The centralized nightlies job failed on Tuesday (2025-06-17)

The centralized nightlies job failed on Tuesday (2025-06-17) in run [15722476549](https://github.com/TileDB-Inc/centralized-tiledb-nightlies/actions/runs/15722476549)

Also, starting on [Monday night](https://github.com/TileDB-Inc/centralized-tiledb-nightlies/actions/runs/15988433703), 3 tiledbsoma-py tests started [failing](https://github.com/TileDB-Inc/centralized-tiledb-nightlies/actions/runs/15988433703/job/45097983097#step:12:6754):

```python
=========================== short test summary info ============================
FAILED apis/python/tests/test_clib.py::test_get_soma_type_metadata_value_from_array_error - tiledb.libtiledb.TileDBError: [TileDB::Array] Error: Cannot open array; Array does not exist.
FAILED apis/python/tests/test_clib.py::test_get_soma_type_metadata_value_from_group_error - tiledb.libtiledb.TileDBError: Group: Cannot open group; Group does not exist.
FAILED apis/python/tests/test_collection.py::test_collection_repr[False] - tiledb.libtiledb.TileDBError: Group: Cannot open group; Group does not exist.
= 3 failed, 7569 passed, 30 skipped, 2 xfailed, 786 warnings in 804.26s (0:13:24) =
```

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -559,10 +559,13 @@ jobs:
           # Fail if not properly linked to nightly libtiledb.so
           ldd builddir/storage/mytile/ha_mytile.so | \
             grep -q $(pwd)/install-libtiledb/lib/libtiledb.so
-      - name: Test TileDB-MariaDB
-        run: |
-          cd builddir
-          ./mysql-test/mysql-test-run.pl --suite=mytile
+      # Failing tests are being tracked upstream at
+      # https://github.com/TileDB-Inc/TileDB-MariaDB/issues/393
+      #
+      # - name: Test TileDB-MariaDB
+      #   run: |
+      #     cd builddir
+      #     ./mysql-test/mysql-test-run.pl --suite=mytile
   tiledb-go:
     runs-on: ubuntu-24.04
     needs: [libtiledb]

